src/views/preview/presentation.jsx

```diff
@@ -136,6 +139,18 @@ const PreviewPresentation = ({
                 message={embedCensorMessage(visibilityInfo.message)}
             />);
         }
+    } else if (justRemixed && isNewScratcher) {
+        banner = (
```

Comment:
Should the just remixed banner show for all users, or just new Scratchers? it looks like it'll only be newScratchers

Reply:
You're right, checked with designers, fixed